specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/examples/ApplicationGroup/ApplicationGroupCreate.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parameters": {
+    "namespaceName": "sayan-ua-test-eh-system-1",
```

Comment:
fix: we should not be using our own name in the swagger specs. These examples are published in our public facing docs. Using a universal name like "contoso" which is created for these purposes.